test/sqllogictest/table.slt

```diff
@@ -0,0 +1,37 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Tests for the esoteric TABLE statement.
+
+mode cockroach
```

Comment:
Add test for `TABLE t ORDER BY a LIMIT 1`.

Reply:
Done.

Reply:
Er, sorry, will be done shortly.

Reply:
Ok, done now.